perun-core/src/main/java/cz/metacentrum/perun/core/entry/ConsentsManagerEntry.java

```diff
@@ -36,4 +46,88 @@ public PerunBl getPerunBl() {
 	public void setPerunBl(PerunBl perunBl) {
 		this.perunBl = perunBl;
 	}
+
+	@Override
+	public List<Consent> getAllConsents(PerunSession sess) throws PrivilegeException {
+		Utils.checkPerunSession(sess);
+
+		// auth
+		if (!AuthzResolver.authorizedInternal(sess, "getAllConsents_policy")){
+			throw new PrivilegeException("getAllConsents");
+		}
+
+		return consentsManagerBl.getAllConsents(sess);
+	}
+
+	@Override
+	public List<Consent> getConsentsForConsentHub(PerunSession sess, int id, ConsentStatus status) throws PrivilegeException {
+		Utils.checkPerunSession(sess);
+
+		getConsentsManagerBl().checkConsentHubExists(sess, consentsManagerBl.getConsentHubById(id));
+
+		// auth
+		if (!AuthzResolver.authorizedInternal(sess, "getConsentsForConsentHub_Id_ConsentStatus_policy")){
```

Comment:
as we'd like to resolve it also for facilityadmins, we'll need to send facility object to the method too. i wrote a suggestion how to resolve it to perun-roles file. i think we won't need to filter it afterwards